src/web/app/pages-session/session-submission-page/__snapshots__/session-submission-page.component.spec.ts.snap

```diff
@@ -2329,6 +2329,7 @@ exports[`SessionSubmissionPageComponent should snap with feedback session questi
                     
                     
                     
```

Comment:
Extra lines added? Please remove if it is not essential

Reply:
These extra lines were generated after updating the snapshot tests; it seems that the number of lines corresponds to the number of types of error messages.